src/Resources/contao/library/Contao/Widget.php

```diff
@@ -263,6 +265,22 @@ public function __set($strKey, $varValue)
 				$this->strPrefix = $varValue;
 				break;
 
+			case 'useRawRequestData':
+				if ($varValue === true)
+				{
+					/** @var Request $request */
+					$request = \System::getContainer()->get('request_stack')->getCurrentRequest();
+					$this->setInputCallback(function() use ($request) {
+						return $request->request->get($this->name);
```

Comment:
I have no idea where this array syntax is needed, maybe @aschempp can help us here? See [Widget.php:781](https://github.com/contao/core-bundle/blob/4758be406700f7861bdddfb39e140de1e61a6b73/src/Resources/contao/library/Contao/Widget.php#L781)